File: app/src/main/java/com/callmangement/EHR/models/ModelLogin.java
package com.callmangement.EHR.models;

import com.google.gson.annotations.SerializedName;

public class ModelLogin {
    @SerializedName("status")
    private String status;
    @SerializedName("message")
    private String message;
    @SerializedName("user_details")
    ModelLoginData user_details = new ModelLoginData();

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public ModelLoginData getUser_details() {
        return user_details;
    }

    public void setUser_details(ModelLoginData user_details) {
        this.user_details = user_details;
    }
}

File: app/src/main/java/com/callmangement/ehr/models/ModelLogin.kt
package com.callmangement.ehr.models

import com.google.gson.annotations.SerializedName

class ModelLogin {
    @JvmField
    @SerializedName("status")
    var status: String? = null

    @JvmField
    @SerializedName("message")
    var message: String? = null

    @JvmField
    @SerializedName("user_details")
    var user_details: ModelLoginData = ModelLoginData()
}
